### PR TITLE
completed TODO task

### PR DESCRIPTION
**Pull request by shalini-tiwari**

### CODE REVIEW

**Review by DeepNinja07x — APPROVED** [review]

File: Scripts/TODO_Scripts/PyCalculator.py [hunk 1]
```diff
@@ -14,33 +14,70 @@ def multiply(num1, num2):
...
+	return math.tan(num1)
+
+#Function to take one input 
+def one_input():
+    num = int(input(:Enter one number: "))
+	return num
+
+#Function to take two input 
+def two_input():
+    num_1 = int(input(:Enter one number: "))
```
Comment (subhayu99) [inline Scripts/TODO_Scripts/PyCalculator.py]:
```suggestion
    num_1 = int(input("Enter one number: ")) 
```

File: Scripts/TODO_Scripts/PyCalculator.py [hunk 2]
```diff
@@ -14,33 +14,70 @@ def multiply(num1, num2):
...
+def cosnum(num1):
+	return math.cos(num1)
+
+# Function to find Tan of a number
+def tannum(num1):
+	return math.tan(num1)
+
+#Function to take one input 
+def one_input():
+    num = int(input(:Enter one number: "))
```
Comment (subhayu99) [inline Scripts/TODO_Scripts/PyCalculator.py]:
```suggestion
    num = int(input("Enter one number: "))
```

File: Scripts/TODO_Scripts/PyCalculator.py [hunk 3]
```diff
@@ -14,33 +14,70 @@ def multiply(num1, num2):
   
 # Function to divide two numbers 
 def divide(num1, num2): 
-    return num1 / num2   
+    return num1 / num2 
+	
+# Function to find Sin of a number
+def sinnum(num1):
+	return math.sin(num1)
```
Comment (subhayu99) [inline Scripts/TODO_Scripts/PyCalculator.py]:
```suggestion
    return math.sin(num1)
```

File: Scripts/TODO_Scripts/PyCalculator.py [hunk 4]
```diff
@@ -14,33 +14,70 @@ def multiply(num1, num2):
...
+
+# Function to find Cos of a number
+def cosnum(num1):
+	return math.cos(num1)
```
Comment (subhayu99) [inline Scripts/TODO_Scripts/PyCalculator.py]:
```suggestion
    return math.cos(num1)
```

File: Scripts/TODO_Scripts/PyCalculator.py [hunk 5]
```diff
@@ -14,33 +14,70 @@ def multiply(num1, num2):
...
+
+# Function to find Tan of a number
+def tannum(num1):
+	return math.tan(num1)
```
Comment (subhayu99) [inline Scripts/TODO_Scripts/PyCalculator.py]:
```suggestion
    return math.tan(num1)
```

**Review by subhayu99 — CHANGES_REQUESTED** [review]
@shalini-kumari2000  Rectify the following syntax errors.

**Comment by subhayu99** [comment]
> Rectify the following syntax errors.

**Review by subhayu99 — APPROVED** [review]
✅

**Pull request merged**